manifold/src/manifold.cpp

```diff
@@ -306,29 +320,37 @@ std::vector<int> Manifold::GetMeshIDs() const {
  *
  * @returns New MeshID
  */
-int Manifold::SetAsOriginal() {
-  int meshID = pImpl_->InitializeNewReference();
-  return meshID;
+std::pair<int, Manifold> Manifold::AsOriginal() const {
```

Comment:
Is it expensive to get the `meshID` anymore? I wonder if it's even worth it to return a pair? It'd be cleaner to just return the manifold if it's easy to extract the `meshID` from that.

Reply:
Yes it should be cheap to get the `meshID`